.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+	// See http://go.microsoft.com/fwlink/?LinkId=827846
+	// for the documentation about the extensions.json format
+	"recommendations": [
+		"platformio.platformio-ide"
+	]
+}
```

Comment:
Newline at end of file

Reply:
deze files zijn auto gegenereerd en worden ignored door git dus maakt niet uit

Reply:
deze worden niet ignored want dan zouden ze niet in je pull request staan.